src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -569,6 +579,23 @@ Apply to thead to make headers not bold.
     margin-bottom: 10px;
 }
 
+.profile-pic-file-selector {
+    overflow: hidden;
+    position: relative;
+}
+
+    .profile-pic-file-selector > #studentPhoto {
+        position: absolute;
+        top: 0;
+        right: 0;
+        min-width: 100%;
+        min-height: 100%;
```

Comment:
It is intended haha. The indentation is used to specify hierarchical definitions and ease of reading